rtd-requirements.txt

```diff
@@ -15,7 +15,11 @@ Sphinx>=4.2.0; python_version >= '3.10'
 docutils>=0.13.1,<0.17; python_version == '2.7'
 docutils>=0.13.1; python_version >= '3.5'
 sphinx-git>=10.1.1
-GitPython>=2.1.1
+
+# GitPython version 3.0.0 and newer does not support python 2.7
+# GitPython version 3.1.27 fixes safety issue 52518
+GitPython>=2.1.1<3.0.0; python_version < 3.0.0
+GitPython>3.1.27; python_version >= 3.0.0
```

Comment:
You need to define the minimum levels and python versions consistently to those in minimum-constrains.txt.

Reply:
DONE